### PR TITLE
NATIVE-238: Display POIs

### PR DESCRIPTION
**Pull request by TechPiranja**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **NATIVE**.

### CODE REVIEW

File: src/routes/dashboard/components/Dashboard.js [hunk 1]
```diff
@@ -99,6 +102,19 @@ class Dashboard extends React.Component<PropsType> {
       }))
     }
 
+    tiles.push(new TileModel({
```
Comment (maxammann) [inline src/routes/dashboard/components/Dashboard.js:105]: This should be disabled before merging this

File: src/routes/pois/components/Pois.js [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    this.props.navigateToPoi({ cityCode, language, path })
+  }
+
+  renderPoiListItem = (cityCode: string, language: string) => (poi: PoiModel) => {
+    const { theme } = this.props
+    return <PoiListItem key={poi.path}
+                          poi={poi}
+                          language={language}
+                          theme={theme}
+                          navigateToPoi={this.navigateToPoi(cityCode, language, poi.path)} />
```
Comment (maxammann) [inline src/routes/pois/components/Pois.js:64]:
```suggestion
                          navigateToPoi={this.props.navigateToPoi({ cityCode, language, path: poi.path })} />

```

File: src/routes/pois/components/Pois.js [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+  navigateToIntegreatUrl: NavigateToIntegreatUrlParamsType => void
+|}
+
+/**
+ * Displays a list of events or a single event, matching the route /<location>/<language>/events(/<id>)
+ */
+class Pois extends React.Component<PropsType> {
+  navigateToPoi = (cityCode: string, language: string, path: string) => () => {
+    this.props.navigateToPoi({ cityCode, language, path })
+  }
```
Comment (maxammann) [inline src/routes/pois/components/Pois.js:56]:
```suggestion
```

File: locales/locales.json [hunk 4]
```diff
@@ -307,7 +307,8 @@
       "offers": "Angebote",
       "events": "Veranstaltungen",
       "news": "Nachrichten",
-      "newsAlternative": "Neuigkeiten"
+      "newsAlternative": "Neuigkeiten",
+      "pois": "Points of Interest"
```
Comment (maxammann) [inline locales/locales.json:311]: These need to be added to the https://github.com/Integreat/integreat-locales/ project. After that you can pull in the locales from there.

File: src/routes/pois/components/Pois.js [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+
+    const createFeedbackVariant = (
+      label: string, feedbackType: FeedbackType, feedbackCategory: FeedbackCategoryType, pagePath?: string
+    ): FeedbackVariant => new FeedbackVariant(label, language, cityCode, feedbackType, feedbackCategory, pagePath)
+
+    const cityTitle = CityModel.findCityName(cities, cityCode)
+    navigation.navigate('FeedbackModal', {
+      isPositiveFeedback,
+      feedbackItems: [
+        createFeedbackVariant(t('feedback:contentOfPoi', { poi: poi.title }), PAGE_FEEDBACK_TYPE,
```
Comment (maxammann) [inline src/routes/pois/components/Pois.js:78]: This key is probably missing in the locales.json

File: src/routes/pois/components/Pois.js [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+
+/**
+ * Displays a list of events or a single event, matching the route /<location>/<language>/events(/<id>)
+ */
+class Pois extends React.Component<PropsType> {
+  navigateToPoi = (cityCode: string, language: string, path: string) => () => {
+    this.props.navigateToPoi({ cityCode, language, path })
+  }
+
+  renderPoiListItem = (cityCode: string, language: string) => (poi: PoiModel) => {
```
Comment (maxammann) [inline src/routes/pois/components/Pois.js:58]: Minor: You could create here a separate component, which could make this file a little bit better readable. Not sure though.

**Review by maxammann — CHANGES_REQUESTED** [review]

File: src/routes/dashboard/components/Dashboard.js (hunk 1 above)
Comment (klinzo) [reply src/routes/dashboard/components/Dashboard.js:105]: There is a flag in the build config that should be used.

File: src/routes/pois/components/Pois.js (hunk 2 above)
Comment (TechPiranja) [reply src/routes/pois/components/Pois.js:66]: @maxammann  i tried to apply both of your changes to `Pois.js` , but they are not working this way.
I also tried to fix it, but my typescript skills are maybe not enough.
I get the error that the return type is not correct !?
Should we leave it the old way? Or can you provide me a proper fix? 🧐

File: locales/locales.json (hunk 4 above)
Comment (TechPiranja) [reply locales/locales.json:311]: ok, can i just revert my changes for locales.json? @maxammann

Comment (Taggotty) [reply locales/locales.json:311]: yes just revert :)

File: src/routes/pois/components/Pois.js (hunk 2 above)
Comment (maxammann) [reply src/routes/pois/components/Pois.js:65]: lets leave it the old way :)

**Review by maxammann — APPROVED** [review]

**Pull request merged**